DQM/GEM/interface/GEMDQMBase.h

```diff
@@ -615,4 +615,10 @@ inline int GEMDQMBase::getDetOccXBin(const int chamber, const int layer, const i
   return n_chambers * (chamber - 1) + layer;
 }
 
+inline Float_t GEMDQMBase::restrictAngle(const Float_t fTheta, const Float_t fStart) {
```

Comment:
Hi @perrotta, 
Sorry for my late...

And that's what I'm looking for. Thanks!
But I also agree that the current one is more readable, so I'll keep it. ;) However, I would use your useful suggestion in some other place rather than copy `restrictAngle()`.

Best regards, 
Byeonghak Ko